mewarpx/mewarpx/utils_store/parallel_util.py

```diff
@@ -22,6 +18,7 @@ def mpiallreduce(data=None, op=None, comm=None):
     return result
 
 def parallelsum(a, comm=None):
+    from mewarpx.mwxrun import mwxrun
```

Comment:
The reason mwxrun was imported for the whole file rather than this function is that there are other functions in warp's parallel_util that check if there is more than 1 processor `mwxrun.n_procs <= 1`.

Reply:
So I moved it because if it's imported for the whole file the simulation crashes. Figuring that out is what delayed this PR a little.

Reply:
Oh wow that's weird...